docs/source/tutorials/advanced/backbones.rst

```diff
@@ -1,7 +1,142 @@
+    |   3   |     mmdet.ResNetV1d     |     depth     |      18, 34, 50, 101, 152     |
+    |   4   |      mmdet.ResNeXt      |     depth     |          50, 101, 152         |
+    |   5   |       mmdet.SSDVGG      |   input_size  |            300, 512           |
+    |       |                         |     depth     |           11, 16, 19          |
+    |   6   |       mmdet.HRNet       |     extra     |                               |
+    |   7   |      mmdet.Res2Net      |     depth     |          50, 101, 152         |
+    |   8   |  mmdet.DetectoRS_ResNet |     depth     |          50, 101, 152         |
+    |   9   | mmdet.DetectoRS_ResNeXt |     depth     |          50, 101, 152         |
+    |   10  |      mmdet.Darknet      |               |                               |
+    |   11  |      mmdet.ResNeSt      |     depth     |       50, 101, 152, 200       |
+    |   12  |     mmdet.CSPDarknet    |               |                               |
+    +-------+-------------------------+---------------+-------------------------------+
+
+3. We need to run the command below to replace the backbone.
+
+  In this example, we'll replace the classification model using the default EfficientNet with mmdet.ResNet (use Backbone Type).
+
+  .. code-block::
+
+    (otx) ...$ otx build --backbone mmdet.RegNet
+
+    [*] Backbone Config: mmdet.RegNet
+    [*] mmdet.RegNet requires the argument : ['arch']
+    [*] Please refer to /venv/lib/python3.9/site-packages/mmdet/models/backbones/regnet.py
+    [*] 'arch' can choose between: ['regnetx_400mf', 'regnetx_800mf', 'regnetx_1.6gf', 'regnetx_3.2gf', 'regnetx_4.0gf', 'regnetx_6.4gf', 'regnetx_8.0gf', 'regnetx_12gf']
+    [*] 'arch' default value: regnetx_400mf
+    [*] Save backbone configuration: otx-workspace-CLASSIFICATION/backbone.yaml
+    [*] Update model.py with backbone.yaml
+            Target Model: SAMImageClassifier
+            Target Backbone: mmdet.RegNet
+            Backbone config: {'arch': 'regnetx_400mf', 'avg_down': False, 'base_channels': 32, 'conv_cfg': None, 'dcn': None, 'deep_stem': False, 'dilations': (1, 1, 1, 1), 'frozen_stages': -1, 'in_channels': 3, 'init_cfg': None, 'norm_cfg': {'requires_grad': True, 'type': 'BN'}, 'norm_eval': True, 'out_indices': (0, 1, 2, 3), 'plugins': None, 'pretrained': None, 'stage_with_dcn': (False, False, False, False), 'stem_channels': 32, 'strides': (2, 2, 2, 2), 'style': 'pytorch', 'type': 'mmdet.RegNet', 'with_cp': False, 'zero_init_residual': True}
+    [*] Save model configuration: model.py
+
+  Then we get model.py, which has been changed to mmdet.ResNet.
+
+  .. note::
+
+    If you get a log like the one below, Follow the steps below.
+
+    .. code-block::
+
+      [!] mmseg.HRNet backbone has inputs that the user must enter.
+      [!] Edit backbone.yaml and run 'otx build --backbone backbone.yaml'.
+
+    Please modify the available configuration file directly (backbone.yaml).
```

Comment:
Maybe it is possible to give an example what exactly should we change to work with such backbones?